src/detail-page/components/charts/Chart.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { ChartSource } from '../../../data/charts/chartInterfaces';
+import { ProfessionId } from '../../../data/home-page/ITProfessions';
+
+interface ChartInterface {
+  maxWidth: number;
+  title: string;
+  source: ChartSource | null;
+  type: ProfessionId;
+}
+
+const StyledWrapper = styled('div')`
+  height: auto;
+  width: 100%;
+`;
+
+const StyledTitle = styled(Typography)`
+  font-weight: bold;
+  margin-bottom: ${({ theme }) => `${theme.spacing(2)}px`};
+`;
+
+const StyledSource = styled('p')`
+  font-size: 0.8rem;
+  margin-top: ${({ theme }) => `${theme.spacing(3)}px`};
+  text-align: right;
+`;
+
+export const Chart: FC<ChartInterface> = ({ children, maxWidth, title, source, type }) => {
```

Comment:
I create component that can share title/source and other element with all charts.

Reply:
I like it a lot! :clap: Also, maybe we could reflect that fact that it's a wrapper in the name and call it e.g. `ChartWrapper`? It doesn't really have any chart logic, because we pass the chart as child :)